### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gamebits**
Adding more details about what the default folders are, with links to relevant documentation.

### CODE REVIEW

**Comment by WPprodigy** [comment]
> `private`: For any files which must not be web-accessible — [more details here](https://wpvip.com/documentation/vip-go/understanding-your-vip-go-codebase/#using-private).

> These directories will also be available on production web servers. Any additional directories created in your GitHub repository that are not included in the above list will not be mounted onto your site, and so will not be web-accessible.

These two statements kind of conflict. Maybe just a little more information on the private folder would be helpful, taken from the docs:

> `private`: Provides access to files that are not directly web accessible, but can be accessed by your theme or plugin code — [more details here](https://wpvip.com/documentation/vip-go/understanding-your-vip-go-codebase/#using-private).

**Comment by gamebits** [comment]
@GaryJones To address @WPprodigy's point, would it be an easy fix to change

> These directories will also be available on production web servers

To:

> With the exception of `private`, these directories will be available on production web servers

File: README.md [hunk 1]
```diff
@@ -18,7 +18,17 @@ To dig straight into our documentation and get up and running, try:
...
+All the following directories are required and must not be removed:
+
+* `client-mu-plugins`: for always active, global plugins (similar to `mu-plugins`) — see [our documentation](https://wpvip.com/documentation/vip-go/managing-plugins/#installing-to-the-client-mu-plugins-directory) for more information.
+* `images`: Store your site's favicons here, per [this documentation](https://wpvip.com/documentation/vip-go/understanding-your-vip-go-codebase/#favicons). All other public-facing images should be uploaded or [imported](https://wpvip.com/documentation/launching-with-vip/content-migration/) to the WordPress dashboard or stored as part of your `/theme/` assets.
+* `languages`: For `.po` and `.mo` translation files, which specify the translated strings for the site.
+* `plugins`: Your site's plugins — more details [here](https://wpvip.com/documentation/vip-go/managing-plugins/#installing-to-the-plugins%C2%A0directory).
+* `private`: For any files which must not be web-accessible — [more details here](https://wpvip.com/documentation/vip-go/understanding-your-vip-go-codebase/#using-private).
```
Comment (WPprodigy) [inline README.md]:
```suggestion
* `private`: Provides access to files that are not directly web accessible, but can be accessed by your theme or plugin code — [more details here](https://wpvip.com/documentation/vip-go/understanding-your-vip-go-codebase/#using-private).
```

**Comment by WPprodigy** [comment]
@gamebits I think that second part is fine as-is. The private directory is available on the production web server. The below is still accurate.

> These directories will also be available on production web servers. Any additional directories created in your GitHub repository that are not included in the above list will not be mounted onto your site, and so will not be web-accessible.

**Pull request merged**